web-marketplace/src/components/organisms/DashboardNavigation/DashboardNavigation.Dropdown.tsx

```diff
@@ -24,14 +29,23 @@ export const AccountSwitcherDropdown = ({
       role="listbox"
       aria-label={_(msg`Account switcher`)}
     >
-      {accounts.map(account => {
-        const isSelected = account.id === activeId;
+      {accounts.map((account, index) => {
+        const accountId = account.id || account.address || String(index);
```

Comment:
why relying on id or address or the index for the accountId?
couldn't we just rely on one of them?